### ISSUE TITLE
[SD3] the training script of SD3 dreambooth has wrong logit-normal weighting

### ISSUE THREAD

**Issue by Luciennnnnnn** [huggingface/diffusers#8534]
### Describe the bug

I guess the implementation of logit-normal weighting is wrong https://github.com/huggingface/diffusers/blob/896fb6d8d7c10001eb2a92568be7b4bd3d5ddea3/examples/dreambooth/train_dreambooth_lora_sd3.py#L1490.

Consider that timestep is sampled uniformly first, weighting function should be a function that depends on pre-sampled timestep, however current version is just a random sample of logit-normal distribution.

### Reproduction

no

### System Info

no

### Who can help?

@DN6 @yiyixuxu @sayakpaul @DN6 cc.

**Comment by sayakpaul** [comment]
Fixed by #8528. Cc: @kashif

**Comment by feng-yufei** [comment]
Request: can anyone confirm that the default sigma_sqrt corresponds to the uniform weighting? After checking with the training objective here (https://github.com/huggingface/diffusers/issues/8567) I found the training loss has an additional sigma, and I wonder whether all the reweighting should be based on the sigma_sqrt first.

**Comment by sayakpaul** [comment]
^ seems like a different issue than the original question. So, will close this issue.
